src/components/Modal/Modal.jsx

```diff
@@ -111,27 +100,40 @@ class Modal extends React.Component {
       loading
     });
 
+    const transitionStyles = {
+      entered: {
+        opacity: 0.95
+      }
+    };
+
     return (
-      <CSSTransitionGroup
-        transitionName="transition"
-        transitionAppear
-        transitionAppearTimeout={300}
-        transitionEnter={false}
-        transitionLeave={false}
+      <Transition
+        in
+        timeout={300}
+        appear
+        onEntered={this.onOpen}
+        onExited={this.onClose}
       >
-        <StyledAside key="modal" className={asideClasses} {...rest}>
-          <header className="header">
```

Comment:
The header element should not be displayed if there is no header prop.

Reply:
There is the same kind of code in `Sidebar`, from which I wrote `Modal`.
In the `<header>`, we display both the title and the header.
We might add the following condition to display the `<header>`: `if (header || title)`.

What do you think ?

Reply:
In the Sidebar the header contains another components: back button, close button

They can be hidden but it's kind of a convention to reserve the space for the buttons and title even when there is nothing. The main reason is to keep the first readable line at the same position, the eye is not lost between two Sidebars when you navigate through them.

The Modal is different in my opinion. It's a one shot display and I can't see the point to reserve the space, especially on mobile. Try to display the Modal on responsive/mobile mode and you'll see a big empty space on top of the Modal. It's kind of weird (just add `iframe.html` between `localhost/` and the query string to get rid of the Storybook UI ;) ).

Reply:
Alright, I'll adapt `Modal` in that sense.